### PR TITLE
Redemption sites fix

### PR DESCRIPTION
**Pull request by cacraig**
Added to slugify command to account for new agency.

### CODE REVIEW

File: ekip/nationalparks/models.py [hunk 1]
```diff
@@ -115,6 +115,16 @@ class FederalSite(models.Model):
...
         return (self.annual_pass and self.active_participant)
 
+    def save(self, *args, **kwargs):
+        """ Generate and save a slug when this object is saved for the first
+        time. """
+
+        if not self.id:
+            name = self.name
+            name = name.replace('National Wildlife Refuge', 'NWR')
+            self.slug = slugify(name + self.city)[:80]
```
Comment (khandelwal) [inline ekip/nationalparks/models.py]: Just name and city don't generate decent enough slugs - especially if you're trying to replace the slugify management command. If you look at that, it does a lot of work to ensure the slugs are meaningful, short and unique.

File: ekip/nationalparks/data/pass-list-20160603.csv [hunk 2]
```diff
@@ -544,8 +544,7 @@ Kisatchie NF - Winn District,318-628-4664,Winnfield,LA,http://www.fs.fed.us/r8/k
 Adams National Historical Park,617-770-1175,Quincy,MA,http://www.nps.gov/adam,YES,YES
 Assabet River NWR,978-562-3527,Sudbury,MA,http://www.fws.gov/refuge/assabet_river/,YES,YES
 Boston National Historical Park,617-242-5642,Boston,MA,http://www.nps.gov/bost,NO,YES
-Buffumville Lake,"508-248-5697
```
Comment (khandelwal) [inline ekip/nationalparks/data/pass-list-20160603.csv:547]: Is this change related to the slugification? If not - maybe it should be a separate PR.

**Comment by cacraig** [comment]
@khandelwal Good point. I removed the automation for slugifying the FederalSite. However, I left in the max_length increase.

**Comment by cacraig** [comment]
@khandelwal - Removed the length increase, and added to the slugify command to account for the new agency.
